### PR TITLE
Interactive schema change

### PR DESCRIPTION
**Pull request by shubhi1092**

### CODE REVIEW

File: app/schemas/models/interactives/interactive.schema.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ const schema = require('../../schemas')
 
 const interactiveSchema = {
   type: 'object',
-  required: ['interactiveType', 'promptText', 'interactiveData', 'unitCodeLanguage'],
```
Comment (AndrewJakubowicz) [inline app/schemas/models/interactives/interactive.schema.js]: Why was this removed?

Comment (shubhi1092) [reply app/schemas/models/interactives/interactive.schema.js]: I created the server APIs keeping the editor in mind. From the editor, I want to allow creating a new interactive with just a name, therefore removed these from the required properties.
